backend/pick-git/src/test/java/com/woowacourse/pickgit/integration/authentication/OAuthServiceIntegrationTest.java

```diff
@@ -142,4 +168,19 @@ void findRequestUserByToken_NotFoundToken_ThrowException() {
         assertThatThrownBy(() -> oAuthService.findRequestUserByToken(token))
             .isInstanceOf(InvalidTokenException.class);
     }
+
+    @DisplayName("authentication이 Null 이라면 GuestUser를 반환한다.")
+    @Test
+    void findRequestUserByToken_NullAuthenticationParam_ReturnGuestUser() {
+        // given
+        // when
+        AppUser appUser = oAuthService.findRequestUserByToken(null);
+
+        // then
+        assertThat(appUser).isInstanceOf(GuestUser.class);
+        assertThatThrownBy(() -> appUser.getUsername())
+            .isInstanceOf(UnauthorizedException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> appUser.getUsername())
            .isInstanceOf(UnauthorizedException.class)
            .hasFieldOrPropertyWithValue("errorCode", "XXX")
            .hasFieldOrPropertyWithValue("httpStatus", "XXX")
            .hasMessage("XXX");
```

이런 식으로 예외 필드 값도 동일한지 확인할 수 있어요!